packages/app-page-builder/src/admin/plugins/permissionRenderer/PageBuilderPermissions/PageBuilderPermissions.tsx

```diff
@@ -49,7 +49,7 @@ export const PageBuilderPermissions = ({ value, onChange }: PageBuilderPermissio
     }, []);
 
     const onFormChange = useCallback(
-        formData => {
+        (formData: any) => {
```

Comment:
Same as my previous comment. Please dont use `any`. Yes, there are some `any` in the code, but we are trying to eliminate it.